### PR TITLE
New Navbar added with the logo of PublicLab

### PR DESCRIPTION
**Pull request by YogeshSharma01**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by govindgoel** [comment]
@YogeshSharma01  Kindly share the preview of changes, is it referred to any issue kindly mention that too.

**Comment by YogeshSharma01** [comment]
Hi, @govindgoel sorry for the late reply actually I was busy with my exams. Here is a short video of changes.



https://user-images.githubusercontent.com/51679200/104451321-610c9700-55c7-11eb-80b7-46eb0f7a2f3b.mov

File: index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <link rel="manifest" href="./manifest.json">
     <script src="./src/scripts/pwainit.js"></script>
 
-
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (HarshKhandeparkar) [inline index.html:24]: Jquery is included in the very next line, I don't think this is needed :)

File: index.html [hunk 2]
```diff
@@ -42,7 +42,56 @@
...
+                <ul>
+                    <li><a class="line" href="https://github.com/publiclab/community-toolbox">Source Code</a></li>
+                    <li><a class="line" href="https://publiclab.org/conduct">Conduct</a></li>
+                    <li><a class="line" href="https://guides.github.com/activities/hello-world/">New To GitHub?</a></li>
+                    <li><a class="line" href="https://github.com/publiclab/plots2/labels/first-timers-only">Selection of
+                            Issues</a></li>
+                    <li><a class="line"
+                            href="https://publiclab.org/notes/warren/11-08-2016/help-public-lab-s-software-grow-by-joining-a-supportive-team">Support</a>
+                    </li>
+                    <li><a class="line" href="https://github.com/publiclab/plots2/labels/help-wanted">Challenges</a>
```
Comment (HarshKhandeparkar) [inline index.html:88]: https://github.com/search?q=label%3Ahelp-wanted+is%3Aissue+is%3Aopen+user%3Apubliclab&ref=simplesearch

This link searches all `help-wanted` issues in all Publiclab repos. (might be useful)

File: index.html [hunk 3]
```diff
@@ -42,7 +42,56 @@
...
+                </ul>
+            </li>
+            <li><a class="line" href="https://publiclab.org/chat">CHATROOM</a></li>
+            <li>
+                <a class="line" href="#">MORE</a>
+                <ul>
+                    <li><a class="line" href="https://github.com/publiclab/community-toolbox">Source Code</a></li>
+                    <li><a class="line" href="https://publiclab.org/conduct">Conduct</a></li>
+                    <li><a class="line" href="https://guides.github.com/activities/hello-world/">New To GitHub?</a></li>
+                    <li><a class="line" href="https://github.com/publiclab/plots2/labels/first-timers-only">Selection of
```
Comment (HarshKhandeparkar) [inline index.html:83]: https://github.com/search?q=label%3Afirst-timers-only+is%3Aissue+is%3Aopen+user%3Apubliclab&ref=simplesearch

This link searches all fto issues in all Publiclab repos. (might be useful)

File: index.html (hunk 1 above)
Comment (YogeshSharma01) [reply index.html]: Yes, I have removed it.

File: index.html (hunk 2 above)
Comment (YogeshSharma01) [reply index.html]: Yes, it is more useful as we can now search for all issues in Publiclab repos.

Comment (YogeshSharma01) [reply index.html]: and I have replaced the link.
Thanks

File: index.html (hunk 3 above)
Comment (YogeshSharma01) [reply index.html]: Yes, I change the link now we can search all fto in all Publiclab repos. 
Thanks

Comment (jywarren) [reply index.html]: very cool, thank you!

File: index.html [hunk 4]
```diff
@@ -42,7 +41,56 @@
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="examples/demo.css">
     <link rel="stylesheet" href="examples/themes.css">
-
+  <!-- Navbar start -->
+    <nav class="navbar">
+        <img id="logo-img" src="./images/Boots-ground-02.png" alt="publiclab-logo">
```
Comment (jywarren) [inline index.html:46]: Let's make this a tiny bit smaller -- can we do... half this size? Also, i don't see this image file, do you need to include it? https://github.com/publiclab/community-toolbox/tree/main/images

**Review by jywarren — CHANGES_REQUESTED** [review]
Thank you, this looks so cool!

File: index.html (hunk 4 above)
Comment (YogeshSharma01) [reply index.html]: Yes, I have set the size of the LOGO-IMG and also add an image file.
Thank you  @jywarren
Logo

**Comment by YogeshSharma01** [comment]
Hi, @jywarren, @govindgoel  I have set the size of the logo image. Please merge the PR.

**Comment by daemon1024** [comment]
Hi @YogeshSharma01 , thanks for the awesome work on this. I had a few pointers,
image
1. The navbar background is not consistent with the rest of the page in dark mode.
2. Is it possible to have the `Fork me on github` banner at the top of the page above navbar?

**Comment by jywarren** [comment]
Thank you!!! This is getting there -- i agree with @daemon1024's input, thank you!!

**Comment by YogeshSharma01** [comment]
Hi, 
Thanks, @daemon1024 I have fixed the dark mode issue and also placed Fork me on GitHub banner at the top of the page above the navbar.
DAY Mode>>
Screenshot 2021-01-30 at 12 03 44 AM
NIGHT Mode>>
Screenshot 2021-01-30 at 12 04 00 AM

**Comment by jywarren** [comment]
This is really lovely!!! Thank you!! I'll go ahead and merge this now!!!

**Comment by daemon1024** [comment]
Fixed #319

**Comment by jywarren** [comment]
Hi @YogeshSharma01 just a quick query - I'm testing this out in GitPod and seeing the following. I ran `npm install` -- am I missing some step to get the navbar formatting?

image

**Comment by YogeshSharma01** [comment]
Hi, @jywarren  I debugged the code and found out this is an issue. The issue was introduced while resolving conflicts in the PR https://github.com/publiclab/community-toolbox/pull/422. 
These are the issues that have found out..

1. There is duplicate code for the navbar and the headings of **FTO**  and **Contributors**  sections.
2. CSS file had a missing parenthesis due to which the whole page was misaligned.
these issues can be resolved easily. So, what do you suggest how do we push the changes?

**Comment by jywarren** [comment]
Hmm. Thank you so much for diagnosing this. If you are able to push a new
PR i'd appreciate it, otherwise if you could open an issue describing this,
we can see if someone else is able to make a fix. Thanks again!!!

**Comment by YogeshSharma01** [comment]
Hi, @jywarren my apologies for the late reply actually because of some medical emergency I wasn't able to reply ok I'll create a new issue for this. 
Thanks

**Comment by jywarren** [comment]
I'm so sorry to hear that and best wishes. Please don't worry too much about this and be in touch when you're able to. ❤️

**Pull request merged**